code/css/style.css

```diff
@@ -0,0 +1,198 @@
+/* style.css */
+
+/* setup */
+* {
+  box-sizing: border-box;
+}
+
+/* clear fix hack */
+.clearfix:after {
+  content: ".";
+  display: block;
+  clear: both;
+  line-height: 0;
+  height: 0;
+}
+
+.clear {
+  clear:both;
+}
+
+.alignleft {
+  float: left;
+}
+
+.alightright {
+  float: right;
+}
+
+/******************************************
+/* BASE STYLES
+/*******************************************/
+
+/* lyout */
+.container {
+  width: 960px;
+  margin: 0 auto;
+}
+
+body {
+  color: ;
```

Comment:
Whoop - careful here. Gotta fill in every css property with a value. If you're not using it, make sure to remove it.